sds_data_manager/stacks/indexer_lambda_stack.py

```diff
@@ -145,7 +145,7 @@ def __init__(
                 detail_type=["Batch Job Started"],
                 detail={
                     "file_to_create": [{"exists": True}],
-                    "status": ["InProgress"],
+                    "status": ["INPROGRESS"],
```

Comment:
Related to another comment - should this use the status ENUM?

Reply:
I am having scope/access issue. Can I revisit this in future PR?